### ISSUE TITLE
When the "ignore-policies" option is enabled, GitHub Action disregards the "extra" parameter

### ISSUE THREAD

**Issue by dimapasko** [Legit-Labs/legitify#247]
### TL;DR / Expected behavior

GitHub Action should be designed to accommodate both parameters, namely "extra" and "ignore-policies".

### Version

1.0.1/main

### On which operating system are you using legitify?

Linux

### Relevant log output

```text
execute legitify analyze: [
  'analyze',
  '--org',
  'scalepoint',
  '--ignore-policies-file',
  './ignored-policies',
  '--output-format',
  'json',
  '--output-file',
  'legitify-output.json'
]
```

### Additional information / Potential fix

I believe the problem lies within this specific line: https://github.com/Legit-Labs/legitify/blob/main/index.js#L162
It seems that the function exits before collecting the `extra` parameter.

These lines could also potentially contribute to the issue:
	•	https://github.com/Legit-Labs/legitify/blob/main/index.js#L147
	•	https://github.com/Legit-Labs/legitify/blob/main/index.js#L153

**Comment by noamd-legit** [comment]
Thanks @dimapasko good catch!
Could you check if this branch fix your problem https://github.com/Legit-Labs/legitify/pull/new/fix/action-extra-param?

**Comment by noamd-legit** [comment]
It should work now.
The extra param needed to be broken to the specific additional parameters.

And yes please open a different issue.

**Comment by noamd-legit** [comment]
merged, closing the issue
